pandas_vet/__init__.py

```diff
@@ -128,6 +129,18 @@ def check_for_unstack(node: ast.Call) -> List:
     return []
 
 
+def check_for_read_table(node: ast.Call) -> List:
+    """
+    Check AST for occurence of the `.read_table()` method on the pandas object.
+
+    Error/warning message to recommend use of `.read_csv()` method instead.
+    """
+    errors = []
```

Comment:
Match the new pattern that avoids `errors.append`

```suggestion
    if isinstance(node.func, ast.Attribute) and node.func.attr == "read_table":
        return [PD012(node.lineno, node.col_offset)]
    return []
```